dockerfile_build/Dockerfile.ubuntu_build

```diff
@@ -0,0 +1,7 @@
+FROM bazel/reproducible:ubuntu
+RUN apt-get update && apt-get -y -q install build-essential python2.7
```

Comment:
You probably want to add an && apt-get clean && rm -rf ... like we have here:
https://github.com/GoogleCloudPlatform/nodejs-docker/blob/master/runtime-image/Dockerfile#L15

Reply:
done